resources/translations/en.edn

```diff
@@ -227,11 +227,13 @@
   :create-resource {:licenses-selection "Licenses"
                     :resid "Resource identifier"
                     :resid-placeholder "urn:test:fi:some-uuid"}
-  :create-workflow {:auto-approve-workflow "Auto-approve"
-                    :auto-approve-workflow-description "The application is approved immediately without human intervention."
+  :create-workflow {:bureaucratic-workflow "Bureaucratic workflow"
```

Comment:
The most exact name would be to name the workflows after the organization which uses them. But since we want to keep customer names out of open source code, the workflow definitions could be moved to the config file.

Reply:
How about "default" workflow" and "decider workflow"?